website/docs/components/table/partials/content/null-values.md

```diff
@@ -0,0 +1,61 @@
+## Null values
+
+### Null cell values
+
+If records within a table contain empty or null values, don’t reflect this literally with an empty cell. While a literal representation of a data set may seem logical when showcasing tabular data, a null value still intrinsically has an attribute of `none` or `empty` which should be communicated to user.
+
+An empty cell can impact the user experience negatively by:
+
+- Breaking the natural reading flow within the table and making the data harder to parse.
+- Eroding user trust in the validity of the data; an empty cell may indicate an error but doesn’t communicate what the error is or its cause.
+- Failing to communicate what value is used when filtering or sorting a data set.
+
+!!! Dont
+
+![Null empty cells](/assets/components/table/null-empty-cells.png)
+!!!
+
+Instead, explicitly communicate null values to the user and represent them with a similar visual treatment as non-null values.
+
+!!! Do
+
+Visually represent null values in an inverse and comparative manner with non-null values.
+
+![Null value communicated with text](/assets/components/table/null-value-comparative-value.png)
+!!!
+
+#### Styling null values
+
+In records that express a value with text, use the same text style as non-null values in the same column (in most cases this is `Body / 200 / Regular`). Consider reducing the prominence of the null values by using `Foreground / Faint` color instead of `Primary` or `Strong`.
+
+![Null value in a text string](/assets/components/table/null-value-text-example.png)
+
+#### Null values with badges
+
+In records that express a value with a badge (e.g., status, health, etc), maintain consistency with cells of the same content type by communicating null values in a `neutral` badge.
```

Comment:
This sentence is a bit difficult for me to parse. Is there a clearer, simpler way to phrase this?

What are you trying to say? Do you mean that we recommend using a `neutral` badge for null content?